client/my-sites/domains/domain-management/dns/index.jsx

```diff
@@ -20,20 +21,20 @@ import SectionHeader from 'components/section-header';
 import DnsTemplates from '../name-servers/dns-templates';
 import VerticalNav from 'components/vertical-nav';
 
-const Dns = React.createClass( {
-	propTypes: {
+class Dns extends React.Component {
+	static propTypes = {
 		domains: React.PropTypes.object.isRequired,
 		dns: React.PropTypes.object.isRequired,
 		selectedDomainName: React.PropTypes.string.isRequired,
 		selectedSite: React.PropTypes.oneOfType( [
 			React.PropTypes.object,
 			React.PropTypes.bool
 		] ).isRequired
-	},
+	};
 
-	getInitialState() {
-		return { addNew: true };
-	},
+	state = {
+		addNew: true
+	};
 
 	renderDnsTemplates() {
 		const selectedDomain = getSelectedDomain( this.props );
```

Comment:
Glad you caught this. Sorry about that. Seems that AnnaMag noticed it here: https://github.com/AnnaMag/wp-calypso/commit/7d6722647b21cc9f257821213cf774249e6a8097 But somehow I didn't see that commit to my (already merged PR). 😢